repository/expression.js

```diff
@@ -45,10 +45,19 @@ class ExpressionRepository {
     });
   }
 
-  deleteExpressionFromUsers(expressionId) {
+  deleteExpressionFromUser(expressionId, userId) {
```

Comment:
Here as well. You can name this `deleteUserExpressionByExpressionIdAndUserId`. Don't be afraid to be explicit, but not too explicit :)